<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="SVMParameters">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>341</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>SVM参数</string>
    </property>
    <widget class="QGroupBox" name="trainingResult">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>训练结果</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>140</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="trainingCountText_label">
           <property name="text">
            <string>训练集大小：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="trainingCount_label">
           <property name="text">
            <string>1090213</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="validationCountText_label">
           <property name="text">
            <string>验证集大小：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="validationCount_label">
           <property name="text">
            <string>123</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="validationAccuracyText_label">
           <property name="text">
            <string>验证集准确率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="validationAccuracy_label">
           <property name="text">
            <string>12%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="timeCounterText_label">
           <property name="text">
            <string>训练总时间(ms)：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeCounter_label">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>73</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="C_label">
        <property name="text">
         <string>惩罚系数C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="C_spinBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>85</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="kernel_label">
        <property name="text">
         <string>Kernel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="kernel_comboBox">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>73</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="gamma_label">
        <property name="text">
         <string>gamma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="gamma_spinBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="startTrainButton">
        <property name="text">
         <string>开始训练</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>73</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="degree_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>多项式度数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="degree_spinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>371</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QComboBox" name="filesSelection_comboBox"/>
       </item>
       <item>
        <widget class="QListWidget" name="allFiles"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="originalImage">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../bin/platecharsamples/chars/_3/00800_颜色法_5_指数_13.jpg</pixmap>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="HOGImage">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../bin/platecharsamples/chars/_3/00800_颜色法_5_指数_13.jpg</pixmap>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="imageLabel">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="predictedImageLabel">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="openFolder_2">
     <property name="title">
      <string>打开训练文件夹</string>
     </property>
     <addaction name="openPlateChar"/>
     <addaction name="openPlateCategory"/>
    </widget>
    <addaction name="openFolder_2"/>
    <addaction name="closeFolder"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="closeFolder">
   <property name="text">
    <string>关闭文件夹</string>
   </property>
  </action>
  <action name="openPlateChar">
   <property name="text">
    <string>训练车牌字符</string>
   </property>
  </action>
  <action name="openPlateCategory">
   <property name="text">
    <string>训练车牌类型</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>on_openFolder()</slot>
  <slot>on_currentFilesItemChanged()</slot>
 </slots>
</ui>
